protected/views/ethnicGroup/_view.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/* @var $this EthnicGroupController */
+/* @var $data EthnicGroup */
+?>
+
+<div class="view">
+
+	<b><?php echo CHtml::encode($data->getAttributeLabel('id')); ?>:</b>
+	<?php echo CHtml::link(CHtml::encode($data->id), array('view', 'id'=>$data->id)); ?>
+	<br />
+
+	<b><?php echo CHtml::encode($data->getAttributeLabel('name')); ?>:</b>
+	<?php echo CHtml::encode($data->name); ?>
+	<br />
+
+	<b><?php echo CHtml::encode($data->getAttributeLabel('code')); ?>:</b>
+	<?php echo CHtml::encode($data->code); ?>
+	<br />
+
+	<b><?php echo CHtml::encode($data->getAttributeLabel('display_order')); ?>:</b>
+	<?php echo CHtml::encode($data->display_order); ?>
```

Comment:
I don't think the display order should be displayed either. That said, I'm not sure how we will work around this. I'll let this one slide until Liam returns. Then we can discuss it with him.

Reply:
The display order text should be removed from this screen. Just leave the editable text field in _form in place.